tsconfig.json

```diff
@@ -35,6 +35,7 @@
         "noUnusedParameters": false,
         "noImplicitReturns": true,
         "noFallthroughCasesInSwitch": true,
+        "skipLibCheck": true,
```

Comment:
What does this change?

Reply:
Probably a better explanation than I could come up with: 
https://stackoverflow.com/questions/52311779/usage-of-the-typescript-compiler-argument-skiplibcheck

> You could find two libraries define the same types, but incompatibly. 

It essentially skips checking ts declaration files, including in imported libs, which may often declare conflicting types or mis-matched versions of types. 

`skipLibCheck` is currently enabled in `mattermost-webapp`.

Reply:
Makes sense. I didn't realize it was true in the web app as well. It seems like one of those things that probably shouldn't be enabled in ideal circumstances, but we're pretty much never writing code in ideal circumstances